CMakeLists.txt

```diff
@@ -61,6 +61,10 @@ endif()
 # PIC (Position Independent Code) ensures .a files can be linked to executables that have PIE enabled
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+if (DO_DISABLE_EXCEPTIONS)
```

Comment:
>DO_DISABLE_EXCEPTIONS [](http://example.com/codeflow?start=4&length=21)

Don't we have to add this as an option above like DO_BUILD_TESTS?

Reply:
Oh, we should do what we did for DO_BUILD_TESTS above, but it's not actually required to set it in the build. I think the above is to just set it as a cmake option

Reply:
Ok I'd still expose it as an option so anyone can see at a glance what configurations we support.